pkg/function/configs.go

```diff
@@ -12,3 +12,19 @@ const (
 	WinStartTime          = "x-numaflow-win-start-time"
 	WinEndTime            = "x-numaflow-win-end-time"
 )
+
+var Protocol string
+var Addr string
+var MAP_MULTIPROC_SERV bool
+
+func init() {
```

Comment:
Instead of using `init()` Move all of these to `client.New()`?

Reply:
Moved